src/components/ToDoContainer/index.js

```diff
@@ -1,11 +1,77 @@
-import React, { Component } from "react";
+import React, {
+  Component
+} from "react";
+
+import NewTodoForm from './NewTodoForm';
+import TodoList from './TodoList';
 
 class ToDoContainer extends Component {
+  constructor() {
+    super();
+    this.state = {
+      newTodo: '',
+      todos: []
+    }
+
+    this.formSubmitted = this.formSubmitted.bind(this);
+    this.newTodoChanged = this.newTodoChanged.bind(this);
+    this.removeTodo = this.removeTodo.bind(this);
+    this.searchTodos = this.searchTodos.bind(this);
+  }
+
+  // Create new todo
+  newTodoChanged(event) {
+    this.setState({
+      newTodo: event.target.value
+    })
+
+  }
+
+  // Form submission event handler
+  formSubmitted(event) {
+    event.preventDefault();
+    // Add new todo to array
+    this.setState({
+      newTodo: '',
+      todos: [...this.state.todos, {
+        title: this.state.newTodo,
+        done: false
+      }]
+    })
+  }
+
+  // Remove clicked todolist
+  removeTodo(index) {
+    const todos = [...this.state.todos];
+    todos.splice(index, 1);
+    this.setState({
+      todos
+    })
+  }
+
+  // Search todolist by substring
+  searchTodos(event) {
+    let todos = [...this.state.todos];
+    todos = todos.filter(todo => todo.title.includes(event.target.value) === true)
```

Comment:
`todo.title.includes(event.target.value)` is a boolean value itself, so no need to check equality with `true`